modules/Project/Save.php

```diff
@@ -133,4 +133,4 @@
 	$_REQUEST['return_url'] = "index.php?module=Project&action=view_GanttChart&project_id=" . $return_id;
     handleRedirect($return_id,'Project');
 }
-?>
\ No newline at end of file
+?>
```

Comment:
@nadilhassan I can't see your change. When I cloned your repository the ?> is still in the file. Can you please fix your master branch?